gcp_delete_old_images.py

```diff
@@ -57,12 +59,59 @@ def delete_old_images():
     subprocess.run(delete_cmd, check=True)
 
 
+def delete_old_docker_images_helper(delete_images, base_cmd, with_tags=False):
+    if not delete_images:
+        return
+
+    delete_images = [f'{image_name[0]}@{image_name[1]}' for
+                     image_name in delete_images]
+
+    print(f"deleting docker images {'with tags' if with_tags else 'without tags'}:\n" +
+          '\n'.join(delete_images))
+
+    for image in delete_images:
+        # '--delete-tags' flag shouldn't be added when deleting images without
+        # tags to catch unseen errors
+        delete_cmd = base_cmd + ['delete', '--quiet'] + \
+            (['--delete-tags', image] if with_tags else [image])
+        subprocess.run(delete_cmd)
+
+
+def delete_old_docker_images():
+    print('\nDeleting docker images')
+    base_cmd = ['gcloud', 'artifacts', 'docker', 'images']
+    get_images_cmd = base_cmd + ['list',
+                                 '--include-tags', '--format',
+                                 'value[separator=" "](package,version,tags)',
+                                 '--filter', 'createTime < -P2W',
+                                 os.environ['GCP_REPO']]
```

Comment:
I suggest using --output json or something like that, so we have a useful output format to work with.